scrape.py

```diff
@@ -0,0 +1,28 @@
+import urllib2
+
+source = "https://data.consumerfinance.gov/api/views/s6ew-h6mp/rows.csv?accessType=DOWNLOAD"
+tmp_name = "data/cfpb_ccdb.csv"
+
+#url = "http://download.thinkbroadband.com/10MB.zip"
+#file_name = url.split("/")[-1]
+
+u = urllib2.urlopen(source)
```

Comment:
In this case we are doing a direct comparison between Go and Python for our own purposes, and so not relying on libs much.  However we will be using the requests library in general for web stuff.